bibliography.bib

```diff
@@ -66,7 +66,7 @@ @inbook {stock2005weak
 	publisher = {Cambridge University Press},
 	organization = {Cambridge University Press},
 	address = {New York},
-	url = {http://www.economics.harvard.edu/faculty/stock/files/TestingWeakInstr_Stock\%2BYogo.pdf},
+	url = {https://scholar.harvard.edu/files/stock/files/testing\_for\_weak\_instruments\_in\_linear\_iv\s_regression.pdf},
```

Comment:
This doesn't seem quite right with `iv\s_`, make sure it compiles correctly before merging?

Reply:
Fixed in 1c0e5dbf05291dd7bdc048c3506b92d8f1a55494. This URL doesn't even show anywhere in the book given how we currently show citations, but it is good to have this correct with with full LaTeX escaping in case we change how that is done.